src/pages/docs/platform/architecture/access-network.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Access network
+meta_description: "Understand how Ably's access network provides reliable, low-latency connectivity to its platform from anywhere in the world."
+---
+
+The access network serves as the entry point to Ably's realtime infrastructure and is core to the platform's ability to deliver reliable, low-latency messaging services globally.
+
+## Why the access network matters
+
+The access network must ensure that clients can reliably connect to the closest available datacenter, providing optimal performance whilst maintaining resilience against various types of failures. Given that Ably is mission-critical for many applications, the reliability of the access network is essential; any connectivity issues can potentially impact thousands or millions of end users, making the design and operation of the access network a key element of Ably's infrastructure.
+
+## Access network architecture
+
+Ably's access network is designed to provide reliable, low-latency connectivity to the Ably platform from anywhere in the world. The architecture leverages multiple AWS services and custom infrastructure components to ensure resilience, performance, and security.
+
+### Global deployment across multiple regions
+
+Ably operates in multiple regions around the world, with servers distributed across more than 15 physical datacenters within Amazon's cloud infrastructure. This global distribution ensures that clients can connect to a nearby datacenter, minimizing the latency between the client and the entry point to Ably's network.
+
+Each datacenter operates independently, with its own dedicated infrastructure for handling client connections, message processing, and storage. This isolation ensures that issues affecting one datacenter do not propagate to others, maintaining service availability even during regional outages.
+
+The global distribution of datacenters also enables Ably to comply with data residency requirements by keeping data within specific geographic regions when required. This is important for customers operating in regulated industries or jurisdictions with strict data sovereignty laws.
+
+### AWS CloudFront and network load balancers
+
+Client connections to Ably are handled through a combination of AWS CloudFront for global edge distribution and AWS EC2 Network Load Balancers (NLBs) for traffic routing within each region.
+
+AWS CloudFront serves as the primary entry point to Ably's network, with over 175 edge locations globally. When a client attempts to connect to Ably, the request is first routed to the nearest CloudFront edge location. This reduces the public internet transit time, as clients connect to a nearby edge node rather than traversing the entire distance to an Ably datacenter.
```

Comment:
```suggestion
AWS CloudFront serves as the primary entry point to Ably's network, with over 700 edge locations globally. When a client attempts to connect to Ably, the request is first routed to the nearest CloudFront edge location. This reduces the public internet transit time, as clients connect to a nearby edge node rather than traversing the entire distance to an Ably datacenter.
```